LDAR_Sim/src/initialization/leaks.py

```diff
@@ -57,6 +57,7 @@ def generate_leak(program, site, start_date, leak_count, days_active=0, day_ts_b
         'days_active': days_active,
         'volume': None,
         'estimated_volume': None,
+        'estimated_volume_b': None,
```

Comment:
Whats this ?

Reply:
Estimated_volume and estimated_volume_b are used to keep track of the estimated emission rate using the method of 1/2 of time from last survey. It's to allow for people to keep track of 2 different groups of site's estimated emissions. Say you had 2 separate state/province that both require you to keep track of this emissions but eventually the calculations need to be different.